_scholars/jj.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: generic
+title: Justin Jackson
+description: Freshman in highschool
+---
+
+LinkedIN: <justin jackson>
```

Comment:
This needs to be a URL
https://www.linkedin.com/in/justin-jackson-69a094147/
I did it for you. Just copy and paste.